ofnetAgent.go

```diff
@@ -817,7 +827,15 @@ func (self *OfnetAgent) RemoveNetwork(vlanId uint16, vni uint32, Gw string, Vrf
 // AddUplink adds an uplink to the switch
 func (self *OfnetAgent) AddUplink(uplinkPort *PortInfo) error {
 	// Call the datapath
-	return self.datapath.AddUplink(uplinkPort)
+	err := self.datapath.AddUplink(uplinkPort)
+	if err != nil {
+		return err
+	}
+
+	if self.protopath != nil {
+		self.protopath.SetRouterInfo(uplinkPort)
```

Comment:
Well, fatal error would mean that we will run into issues when the fwd-mode change is applied. So, a warning or any other notification to user to mention that we are using a specific member link for uplink should be good enough.